lib/profiles.js

```diff
@@ -89,6 +89,25 @@ class Profiles extends Private {
             data: params
         });
     }
+
+    /**
+     * Get the person_ID for a profile using Email
+     * @param {string} email of the profile to get ID
+     * @returns {Promise<Object|KlaviyoApiError>}
+     */
+    getProfileIdByEmail({ email } = {}) {
```

Comment:
Could consider making this just have one argument `email` rather than using an options object to match the style of the other functions with only one argument.